pyigm/clustering/survey_2d2pcf.py

```diff
@@ -169,35 +251,41 @@ def calc_xi_gg(self, sigma=0, error=None):
         if error == 'bs':
             Nf = len(self.fields)
             err_Wbt = np.zeros((len(self.rbinedges) - 1, len(self.tbinedges) - 1), float)
-            Wgg_sum = np.zeros((len(self.rbinedges) - 1, len(self.tbinedges) - 1), float)
-            Wgg_sum2 = np.zeros((len(self.rbinedges) - 1, len(self.tbinedges) - 1), float)
+            W_sum = np.zeros((len(self.rbinedges) - 1, len(self.tbinedges) - 1), float)
+            W_sum2 = np.zeros((len(self.rbinedges) - 1, len(self.tbinedges) - 1), float)
 
             N = self.Nbt
```

Comment:
to be consistent `self.Nbt` should be `self.Nbs` (and re-define previously accordingly)

Reply:
ok